mlflow/models/model.py

```diff
@@ -344,7 +344,7 @@ def model_id(self) -> str:
 
         :getter: Gets the model ID of the logged model
         """
-        return self._logged_model.model_id
+        return self._logged_model.model_id if self._logged_model else None
```

Comment:
nit: shall we add a test case for `_logged_model = None`?

Reply:
`_logged_model = None` is just for safety. It's should be populated.

Reply:
Oh, is it populated for models logged with MLflow 2x?